app/src/main/res/navigation/nav_garden.xml

```diff
@@ -18,43 +18,21 @@
 <navigation xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    app:startDestination="@+id/garden_fragment">
+    app:startDestination="@+id/view_pager_fragment">
 
     <fragment
-        android:id="@+id/garden_fragment"
-        android:name="com.google.samples.apps.sunflower.GardenFragment"
-        android:label="@string/my_garden_title"
-        tools:layout="@layout/fragment_garden">
+        android:id="@+id/view_pager_fragment"
+        android:name="com.google.samples.apps.sunflower.ViewPagerFragment"
```

Comment:
should this live in the root folder or a subfolder within com.google.samples.apps.sunflower?

Reply:
It's in the same folder as the other Fragments.